core/src/main/java/fr/sncf/osrd/envelope_sim/EnvelopeProfile.java

```diff
@@ -0,0 +1,16 @@
+package fr.sncf.osrd.envelope_sim;
+
+import fr.sncf.osrd.envelope.EnvelopeAttr;
+
+/**
+ * The shape of an envelope part.
+ * Please DO NOT add new items to this enum,
+ * and create new attribute types instead.
+ */
+public enum EnvelopeProfile implements EnvelopeAttr {
+    ACCELERATING,
+    CONSTANT_SPEED,
+    MAINTAIN,
```

Comment:
So here MAINTAIN is when the train is trying to catch up with the constant speed it's supposed to have, and CONSTANT_SPEED is as long as it is able to keep its speed ?

Reply:
I'll just rename to CATCHING_UP